### ISSUE TITLE
With async sagas and reducers how can I access one page's sagas or state inside another page

### ISSUE THREAD

**Issue by vimniky** [mxstbr/react-boilerplate#1088]
Hey ! I have a problem with async reducer and sagas. Let's say we have two pages: Home and About . How can I access sagas and the portion state of About page inside Home pages (sagas and reducers of About page haven't loaded yet, so we don't have access to them). Actually in a real world app it quite common we want to do this.

**Comment by avdeev** [comment]
Include About sagas to Home sagas and export them out.

**Comment by vimniky** [comment]
@avdeev but then you'd  loose the benefits of code splitting. Cause I have lots of pages, and many of them are more or less related to each other.

**Comment by avdeev** [comment]
You loose the benefits if you want to access sagas and the portion state of About page inside Home pages

**Comment by vimniky** [comment]
Thanks ! but that still don't solve my problem. Maybe I should just asynchronously load Page component while synchronously load sagas and reducers which is much more simple and I can always get access to all my app's  sagas and states at any time.  I have tweaked react-boilerplate to support server-rendering so I need to run sagas in order to fetch data on the server and then fill up the initialState before the first request returns. Now I have realized that my app's reducers and sagas won't grow to much, so it's fine to just load them synchronously.
